### PR TITLE
docs: add Security Policy

### PR DESCRIPTION
**Pull request by aladdin-add**

### CODE REVIEW

**Review by KFlash — APPROVED** [review]

**Comment by KFlash** [comment]
@aladdin-add merge this?

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied only to the most recent releases.
+
+## Reporting a Vulnerability
+
+To securely report a vulnerability, please email to kataw@gmail.com.
```
Comment (aladdin-add) [inline SECURITY.md:9]: @KFlash what do you think leaving the email??

Comment (KFlash) [reply SECURITY.md:9]: we don't have that email adr

Comment (aladdin-add) [reply SECURITY.md:9]: your personal email?

Comment (KFlash) [reply SECURITY.md:9]: what about yours?

**Pull request merged**